### DISCUSSION TITLE
Weekly Sync - 5 September 2023

### DISCUSSION THREAD

**Discussion by sabineschaller** [interledger/rafiki#1804] [Weekly Sync]
**Please add agenda items below and give your weekly update in a comment. Thank you!**

## Agenda
* @sabineschaller: we have too many open PRs
* @sabineschaller : @huijing could you please give an update about the docs? Especially mermaid support and graphql plugin?
* @sabineschaller : go through alpha.3 milestone and check if we can finish it before we meet in Cluj
* @mkurapov : mention about using discussions to ... discuss

## Notes
* Release notes will have better information about breaking changes, i.e.:
    * sendAmount -> debitAmount, new fees structure
* @sabineschaller to fix link to Open Payments call

**Comment by sabineschaller** [comment]
I had a short week, I was out last week Wednesday to Friday. 
My update:
* still working on liquidity webhook PR
* looked into DIDs to discuss if we can represent KYC info as verifiable credential

**Comment by raducristianpopa** [comment]
Rafiki
 - merged #1812 and #1807 
 - continue to address fedback on #1795
 - reviewing webhook events view PR

Testnet
- working on: https://github.com/interledger/testnet/issues/677
- merged:
  - https://github.com/interledger/testnet/pull/663
  - https://github.com/interledger/testnet/pull/673
  - https://github.com/interledger/testnet/pull/694
  - https://github.com/interledger/testnet/pull/705

Will work on:
- continue with https://github.com/interledger/testnet/issues/677 - start integrating the Open Payments client
- update the `http-signature-utils` library to fetch `httpbis-digest-headers` from the NPM registry

**Comment by Tymmmy** [comment]
Working on Testnet Exchange button, Display currency exchange conversion on sending, and client side websocket for notifications, and organization stuff.

**Comment by BlairCurrey** [comment]
- worked on adding fee calculation to quote. trying to validate/test handling of cross-currency case. Then I just need to update to latest open payments spec (when its merged) and verify I have everything renamed appropriately.
- addressed a bug in our GQL doc check in CI, but holding off on that since that whole workflow is in flux ATM

**Comment by JoblersTune** [comment]
Completed Webhook event filtering

**Comment by mkurapov** [comment]
- Merged in task 1/3 for auto-peering: https://github.com/interledger/rafiki/pull/1805
    - Put up a related PR: https://github.com/interledger/rafiki/pull/1813 (to update our app shutdown/clean up some files)
- Working on remaining two tasks to hopefully complete it this week

**Comment by dragosp1011** [comment]
Worked on improving testnet backend logs, discussed with @rico191013 how to improve the websocket authorization flow

**Comment by njlie** [comment]
* Opened PR for interaction refactoring: #1797 
* Opened PR to refactor `sendAmount` to `debitAmount`: #1806
* Deep diving on latest GNAP spec version to see if there's anything to update

**Comment by Andrei199991** [comment]
opened PR (undrafted it) for Exchange BE endpoint; will address CRs, get it merged and move on with improving Exchange (we have a couple of improvements in mind for it...)

**Comment by beniaminmunteanu** [comment]
Worked on slow completion of incoming payments [issue](https://github.com/interledger/testnet/issues/501)
opened a PR with a comprehensive description abut what is going on : [PR](https://github.com/interledger/testnet/pull/706)

Code reviews on  current testnet features, and providing support on the websocket drive.

**Comment by domin191013** [comment]
created a PR for socket implementation on testnet, improved socket handshake format. discussed new issue about log enhancement.
